graphql_compiler/query_formatting/common.py

```diff
@@ -13,10 +20,89 @@
 # Public API
 ######
 
+def _check_is_string_value(value):
```

Comment:
Should `_check_is_string_value` live within the public API and, if so, might we want to remove the prefixed `_`? This seems like something that should belong with validators.

Reply:
I think my comments below will cause this function to be only used in one place and in simplified form, at which point it can just get inlined.